src/core/analysis/mcmc/MarginalLikelihoodEstimator.cpp

```diff
@@ -44,7 +45,7 @@ MarginalLikelihoodEstimator::MarginalLikelihoodEstimator(const path &fn, const s
         // Initialize
         std::string commandLine;
     
-        //    RBOUT("Processing file \"" + fn + "\"");
+        RBOUT("Processing file \"" + fn.string() + "\"");
```

Comment:
This line is what causes the stepping stone test not to pass – the actual values of marginal likelihoods stay the same. We can probably just regenerate the expected output.